docs/content/latest/troubleshoot/cluster/replace_master.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Replace a failed YB-Master
+headerTitle: Replace a failed YB-Master
+linkTitle: Replace a failed YB-Master
+description: Steps to replace a failed YB-Master in a YugabyteDB cluster.
+aliases:
+  - /troubleshoot/cluster/replace-master/
+  - /latest/troubleshoot/cluster/replace-master/
+menu:
+  latest:
+    identifier: replace-failed-master
+    parent: troubleshoot-cluster
+    weight: 831
+isTocNested: true
+showAsideToc: true
+---
+
+To replace a failed YB-Master server in a YugabyteDB cluster, follow these steps:
+
+For the steps below, the examples use the following scenario:
+
+- The cluster includes three `yb-master` servers: `M1`, `M2`, `M3`.
+- YB-Master server `M1` failed and needs to be replaced.
+- A new YB-Master server (`M4`) will replace `M1`.
+- The default master RPC port is `7100`
+
+
+1. Start the new (replacement) YB-Master server in standby mode. 
+
+To start `yb-master` in standby mode, set the `--master_addresses` flag to an empty string (`""`).
+
+```sh
+./bin/yb-master --master_addresses=""
```

Comment:
not sure if this might end up being confusing to users, if they already have a bunch of other necessary flags, like placement options, fs_data_dirs, etc

I wouldn't want folks to assume this is all the flags they need to bring the master up, but I'm not sure how we're handling that in other sections?

Reply:
Reworded like it's explained here: https://docs.yugabyte.com/latest/manage/change-cluster-config/#2-start-yb-master-servers

Reply:
IMO, now it's actually a bit more confusing, as we're talking about the flag being set to empty string AND the flag not being set, in the same sentence...which for a simple user, could be 2 different things.

What I was thinking was, the command we highlight should either:
- not include `--master_addresses` -- and then the messaging above should say just don't provide it
- include `--master_addresses=""` -- and the messaging should say set it to empty string

But moreover, the command should maybe also include something like
```
--fs_data_dirs=<your data directories>
[any other flags you would normally pass to this master process]
```

Reply:
Pushed change with `--master_addresses=""`